container-messagebus/src/main/resources/configdefinitions/container-mbus.def

```diff
@@ -2,9 +2,24 @@
 namespace=container.jdisc
 
 #settings for message bus in container
-enabled bool default=false
+
+# Which network port is used
 port int default=0
+
+# Number of connections per target
+numconnectionspertarget int default=1
+
+# Number network threads
+numthreads int default=2
+
+# Optimize for latency, or throughput.
+optimize_for enum {LATENCY, THROUGHPUT} default=LATENCY
```

Comment:
this feels a bit generic

If it only controls tcp_nodelay, we can just be more specific. If we add more code alternatives choosing between latency and throughput later we might not be able to test their effects separately, or they need more configuration than a single flag.

Reply:
I do not want to expose too much details. We should figure out a set that works for all.